src/applications/personalization/profile-2/sass/profile-mobile-subnav.scss

```diff
@@ -2,7 +2,7 @@
   .the-menu {
     // setting to `relative` so the z-index is actually used
     position: relative;
-    z-index: $top-layer;
+    z-index: 1; // the megamenu is at 2 so this must be below that.
```

Comment:
This could make sense to include in our "layers" sass, https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/site-wide/sass/modules/_m-layers.scss. There's a todo there about moving the z-index values back into individual modules but I personally like centralizing the z-index values.

Reply:
You mean just move this
```
.mobile-nav .the-menu {
  position: relative;
  z-index: 1;
}
```
to that file? My counterargument against that is that this isn't a sitewide element so it probably doesn't belong in that file. _BUT_ pointing out that file is super helpful. I can change this to `$middle-layer` instead of `1`. This doesn't need to be behind the `#mega-menu` it just needs to be behind the `#mega-menu`'s `#vetnav`, which is at `$top-layer` on mobile.

Reply:
I think I just wanted to point out the file and see what you think. Your approach is great!